build.gradle.kts

```diff
@@ -82,6 +82,7 @@ dependencies {
 
     // Div
     implementation(enforcedPlatform("com.fasterxml.jackson:jackson-bom:$jacksonVersion"))
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.17.+")
```

Comment:
Versjonen bør kanskje hektes på variabel `jacksonVersion` for å alltid være i sync?